src/test/scala/wdlTools/exec/ExecTest.scala

```diff
@@ -46,4 +47,24 @@ class ExecTest extends AnyFlatSpec with Matchers with Inside {
                                               nullCollectionAsEmpty = true)
     values("row") shouldBe V_Pair(V_String("str1"), V_Pair(V_String("str2"), V_String("str3")))
   }
+
+  it should "evaluate task input struct with optional element" in {
+    val doc = parseAndTypeCheck(execDir.resolve("task_input_struct_optional_element.wdl"))
+    // val wf = doc.workflow.getOrElse(
```

Comment:
I'd remove this commented code, the record will remain in the PR history. thanks!

Reply:
removed. Thanks!